tests/compile-fail/unimplemented.rs

```diff
@@ -0,0 +1,16 @@
+#![feature(custom_attribute)]
+#![allow(dead_code, unused_attributes)]
+
+//error-pattern:literal items (e.g. mentions of function items) are unimplemented
+
+static mut X: usize = 5;
+
+#[miri_run]
+fn static_mut() {
+    unsafe {
+        X = 6;
```

Comment:
So yeah, this might fail for a different reason in the future (mutation of a static), but I'm fine with having it here so we notice when that happens.

Although it would be allowed in versions of Miri not intended for rustc const eval.